application/common/models/Method.php

```diff
@@ -368,6 +368,30 @@ public static function getMethods($employeeId)
         }
     }
 
+    /**
+     * Gets all verified methods for user specified by $employeeId
+     * @param string $employeeId
+     * @return string[]
```

Comment:
Is `string[]` the right return type? I would have thought it would either return instances of some class, or possibly arrays.